optuna_dashboard/ts/components/Preferential/PreferentialAnalytics.tsx

```diff
@@ -8,21 +8,35 @@ import {
   useTheme,
 } from "@mui/material"
 import Grid2 from "@mui/material/Unstable_Grid2"
-import { DataGrid, DataGridColumn } from "../DataGrid"
+import { DataGrid } from "../DataGrid"
 import { BestTrialsCard } from "../BestTrialsCard"
 import { useStudyDetailValue, useStudySummaryValue } from "../../state"
 import { Contour } from "../GraphContour"
 import * as Optuna from "@optuna/types"
 
+import { ColumnDef, createColumnHelper } from "@tanstack/react-table"
+
 export const PreferentialAnalytics: FC<{ studyId: number }> = ({ studyId }) => {
   const theme = useTheme()
   const studySummary = useStudySummaryValue(studyId)
   const studyDetail = useStudyDetailValue(studyId)
 
   const userAttrs = studySummary?.user_attrs || studyDetail?.user_attrs || []
-  const userAttrColumns: DataGridColumn<Optuna.Attribute>[] = [
-    { field: "key", label: "Key", sortable: true },
-    { field: "value", label: "Value", sortable: true },
+  const columnHelper = createColumnHelper<Optuna.Attribute>()
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const columns: ColumnDef<Optuna.Attribute, any>[] = [
+    columnHelper.accessor("key", {
+      header: "Key",
+      footer: (info) => info.column.id,
```

Comment:
[nits]: Footer is not created in the table, is this prop really necessary or not?